public/teach.html

```
@@ -1,319 +1,386 @@
+      <a href="/" class="flex items-center gap-2">
+        <img src="/images/logo.png" alt="Alpha One Labs" class="h-8 w-auto" />
+        <span class="font-bold text-xl text-slate-800">Alpha One Labs</span>
+      </a>
+      <div class="hidden md:flex items-center gap-6 text-sm font-medium text-slate-600">
+        <a href="/" class="hover:text-brand">Activities</a>
+        <a href="/dashboard.html" class="hover:text-brand">Dashboard</a>
+        <a href="/teach.html" class="text-brand font-semibold">Host Hub</a>
+      </div>
+      <div class="flex items-center gap-3">
+        <span class="text-slate-600 text-sm hidden sm:block" id="nav-user"></span>
+        <button onclick="logout()" class="text-slate-400 text-sm hover:text-red-500">Logout</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)